.devcontainer/README.md

```diff
@@ -0,0 +1,85 @@
+- [ ] Check if, on the left pane, you have the NPM Scripts (this may be disabled; look at the "Explorer" tree-dots up-right). There will be scripts inside the `package.json` files. You will basically need to run the `dev:server` and the `dev:frontend`. When the frontend finishes starting, a window browser will open **inside** the VSCode. Still, you can open outside.
+
+:warning: **Important for all developers** :warning:
+
+- [ ] Whe you are using the `NODE_ENV=development` the server will not store the configurations you set for security reasons. Please set the proper config on file `.env.development`. The side-effect if you don't, everytime you restart the server, you will be sent to the "Onboarding" page again.
+
+:warning: **Important for Github Codespaces** :warning:
+
+- [ ] When running the "Server" for the first time, its port will be automatically forward. Read the content of the `.env` file on the frontend folder about this, and make sure the port "Visibility" is set to "Public", so the frontend can reach the backend. Again, this is only needed for developing on Github Codespaces. We appreciate to know if you have a better solution.
+
+### For the Collector:
+
+- [ ] Open any file inside the Collector folder. A new option in the status bar (on the left) will appear, colored yellow, called `install deps`. Click on it whenever you want to be sure the v-env is installed and up-to-date.
+- [ ] To run the Collector in "watch" mode (uses the hotdir as input), click the `watch` option in the status bar. It will open a new terminal and run the collector in "watch" mode. You can also run the collector from the terminal, but you must activate the v-env first if the VSCode doesn't do so automatically. To do so, run `source v-env/bin/activate` inside the collector folder and then run the collector with `python watch.py`.
+- [ ] To run the Collector as a background service for the webapp, click the `run` option in the status bar. It will open a new terminal and run the collector using gunicorn. You can also run the collector from the terminal, but you must activate the v-env first if the VSCode doesn't do so automatically. To do so, run `source v-env/bin/activate` inside the collector folder and then run the collector with `FLASK_APP=wsgi.py gunicorn --timeout 300 --workers 4 --bind 0.0.0.0:8888 wsgi:api`.
+
+### For the Collector again:
+
+If you plan to develop the collector, currently (maybe not in a few months), you'll need to install the `pre-release` versions of the extensions to work as they should since MS is moving features around their extensions.
+
+Make sure you have installed the following extensions as `pre-release`:
+
+- Python (ms-python.python)
+- Pylance (ms-python.vscode-pylance)
+- Pylint (ms-python.pylint)
+- Black formatter (ms-python.black-formatter)
+- Python Debugger (ms-python.debugpy)
+
```

Comment:
Do we still need to support this now that we removed all python code from the codebase?